src/data/details.ts

```diff
@@ -1877,6 +1889,30 @@ export const details: Record<number, ShipInfo> = {
       ],
     },
   },
+  57: {
+    name: "Thousand Sunny - 7th Anniversary Special Model",
+    obtain: "Clear the limited time 7th Anniversary campaign quest",
+    period: [
+      "7th Anniversary Phase 1",
+      "After 7th Anniversary",
+    ],
+    effect: [
+      "[Not confirmed] Reduces Special charge time by 5 turn at start of quest, boosts crew's ATK by 1.5x, and boosts Pirate EXP and Berries earned by 3x",
+      "[Not confirmed] Boosts crew's ATK by 1.2x",
```

Comment:
This info is correct so you can remove the _not confirmed_ for the after effect.